src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Issue28343.cs

```diff
@@ -48,6 +48,18 @@ public void ProgressSpinnerWorksWhenReEnabled()
 			App.ScrollUp("CollectionView");
 			App.WaitForElement("RefreshTriggered");
 		}
+
+#if IOS
+		[Test]
+		public void ProgressSpinnerIsVisibleWhenContentWasReset()
+		{
+			App.WaitForElement("SetToEnabled").Tap();
+			App.ScrollUp("CollectionView");
+			App.Tap("ResetContent");
+			App.ScrollUp("CollectionView");
+			VerifyScreenshot("Issue28343_ProgressSpinnerVisible");
```

Comment:
Mnn, the pending snapshot is already available in the latest build.
![image](https://github.com/user-attachments/assets/100923eb-2374-4231-9fc1-4198e408a70d)

However, because is taking a capture from an animated UIActivityIndicator, I think not seems to be the correct way to test it. Better use queries with XPath to find elements and the Rect to verify that are on screen.